core/src/network_manager/composed_protocol.rs

```diff
@@ -29,11 +33,11 @@ use crate::gossip_api::gossip::{AuthenticatedGossipRequest, AuthenticatedGossipR
 
 use super::error::NetworkManagerError;
 
-/**
- * Constants
- */
+// -------------
+// | Constants |
+// -------------
 
-// The maximum size libp2p should allocate buffer space for
+/// The maximum size libp2p should allocate buffer space for
 const MAX_MESSAGE_SIZE: usize = 1_000_000_000;
```

Comment:
This is not pressing, but down the line, should we keep track of these sorts of constants that are bounded by hardware specs so we can expose them as config options?

I imagine if people are gonna be running their own relayers they may want to tweak this sort of thing without forking & digging through the code

Reply:
Yes definitely, we can get to this down the line like you said, but if you have any ideas now, let me know.

I'm thinking they just make their way into the system config, or a second layer config for tunables